austation/code/modules/recycling/disposal/bin.dm

```diff
@@ -0,0 +1,21 @@
+//  Lets us dump items straight into the bin from bags and other storage items.  If the storage item is empty then we'll just pass it to the normal bin behaviour.
+/obj/machinery/disposal/bin/attackby(obj/item/I, mob/user, params)
+	var/obj/item/storage/B
+	var/static/list/obj/item/storage/store_list = list(
+		/obj/item/storage/part_replacer,  //  RPEDs
+		/obj/item/storage/bag,			  //  Botany bags and serving trays etc
+		/obj/item/storage/backpack		  //  Backpacks
+	)
```

Comment:
You don't type a list of typepaths. This isn't an object list.
Also use typecaches.
```suggestion
	var/static/list/store_list = typecacheof(list(
		/obj/item/storage/part_replacer,
		/obj/item/storage/bag,
		/obj/item/storage/backpack
	))
```